cli/projects/configuration.md

```diff
@@ -69,6 +69,7 @@ will occur.
 
 Project-wide configuration can be defined in this block. All possible settings are described in the following subsections.
 
+<a id="the-terramateconfiggit-block"></a>
```

Comment:
Why are we adding those @AnnuCode ?

Reply:
because the heading includes an inline code, I had to add an anchor tag to link to that heading in another markdown file. Without this, the generated heading removed hyphens and couldn't link to the correct section.

Reply:
That's not correct ... e.g. the following link works and jumps directly to the right headline: https://terramate.io/docs/cli/reference/variables/metadata#stack-metadata

Reply:
e.g.

```md
[foo](./terramate.md#the-terramate-block)
```

should work.. the `<a>` element shouldn't be required at all. we don't use it anywhere

Reply:
you are right, I've corrected the section links. Thanks!